e2e/fixtures/views/MetricsReducerView.ts

```diff
@@ -143,4 +143,18 @@ export class MetricsReducerView extends DrilldownView {
     await this.page.getByTestId('list-controls').getByTestId('data-testid template variable').click();
     await this.page.getByRole('option', { name: sortBy }).locator('span').click();
   }
+
+  async selectPrefixFilters(prefixes: string[]) {
+    await this.getByRole('button', { name: 'Prefix filters' }).click();
+    for (const prefix of prefixes) {
+      await this.page.getByTitle(prefix, { exact: true }).locator('div span').click();
+    }
```

Comment:
Let's be careful about scoping the actions to the sidebar:

```suggestion
    const sideBar = this.getSideBar();
    await sideBar.getByRole('button', { name: 'Prefix filters' }).click();

    for (const prefix of prefixes) {
      await sideBar.getByTitle(prefix, { exact: true }).locator('label').click();
    }
```

Reply:
This change will require to fix `getSideBar`'s implementation:
```ts
  getSideBar() {
    return this.getByTestId('sidebar');
  }
```

Reply:
Done in https://github.com/grafana/metrics-drilldown/pull/347/commits/9439e311e7c8cde17ba1303b49b95cc0ec6523ef ✅